### PR TITLE
Added support for timezone

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@donnapep @stulees please review.

Presentation here:
http://rva.risevision.com/#/PRESENTATION_MANAGE/id=eb971a2c-6ce0-4958-a984-38f2b84e5b4e?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

### CODE REVIEW

File: test/integration/time-date-timezone.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  <script src="../../src/components/web-component-tester/browser.js"></script>
+
+</head>
+<body>
+
+<div id="container"></div>
+
+<div id="messageContainer"></div>
+
+<script src="../data/timedate.js"></script>
```
Comment (stulees) [inline test/integration/time-date-timezone.html]: @rodrigopavezi Curious, this data has `useTimezone: false`. Should we also have an integration test that tests the scenario of `useTimezone: true` and has a `timezone` value other than `""`?

Comment (rodrigopavezi) [inline test/integration/time-date-timezone.html]: @stulees good catch, I wonder how the test is passing. I check that. cheers

**Comment by rodrigopavezi** [comment]
@stulees test fixed. cheers

**Comment by stulees** [comment]
@rodrigopavezi :+1: Did you figure out why the test had passed before?

**Comment by rodrigopavezi** [comment]
@stulees yeap, It was the sinon.useFakeTimers. With it what ever the code does it will always return the same time. cheers

**Comment by stulees** [comment]
@rodrigopavezi Ah makes sense, thanks for clarifying

**Pull request merged**